cms/djangoapps/contentstore/views/item.py

```diff
@@ -1135,10 +1134,12 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
     if is_concise:
         if child_info and len(child_info.get('children', [])) > 0:
             xblock_info['child_info'] = child_info
+        user_partitions = get_user_partition_info(xblock, schemes=['random'], course=course)
```

Comment:
`is_concise` is used when we want the concise version of the xblock info (in-short has very few key-values), and it is only used for the moment in Move modal, and `group name to id` issue also appears when we have concise version, otherwise `user_partitions` won't be used here.